src/App.css

```diff
@@ -16,13 +16,13 @@
 .container {
```

Comment:
Define component-specific styles in separate files.

Also, not able to see all the style changes here. Use `git` to transfer all the changes in this branch and only keep the boilerplate/starting code in the `main` branch.